README.md

```diff
@@ -43,6 +43,13 @@ public function tools()
   'en' => 'English',
   'et' => 'Estonian',
 ],
+
+// Or if you are using optimistdigital/nova-locale-manager package
```

Comment:
Can we change this to:
```
// OR

'locales' => function () {
  return Locale::all()->pluck('name', 'key');
},
```
Because it doesn't have to be only used with nova-locale-manager package.